examples/python/helloworld/greeter_client.py

```diff
@@ -15,23 +15,30 @@
 
 from __future__ import print_function
 
+import argparse
 import logging
 
 import grpc
 import helloworld_pb2
 import helloworld_pb2_grpc
 
 
-def run():
+def run(args):
```

Comment:
If you are comfortable with type annotations, can you help to annotate the input argument and return?